ios/Podfile.lock

```diff
@@ -1806,4 +1806,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: fb5148d06b5fd565313eeabfc040cbba30e78c5a
 
-COCOAPODS: 1.14.3
+COCOAPODS: 1.16.2
```

Comment:
Try `bundle exec pod install` instead.
It might not change this version.

Reply:
Done :)